pptx_blueprint/__init__.py

```diff
@@ -82,6 +84,20 @@ def _find_shapes_in_slide(slide):
 
         return matched_shapes
 
+    def _get_all_shapes(self) -> Iterable[BaseShape]:
+        # Do we need all the shapes? Perhaps we should filter on tags here.
+        all_shapes = [shape for slide in self._presentation.slides for shape in slide.shapes]
+        return all_shapes
+
+    def _copy_tags_to_name(self) -> None:
+        all_shapes = self._get_all_shapes()
+        # This regex matches on tags
+        regex_tag = re.compile(r'\{[\s\w]*\}')
+        for shape in all_shapes:
+            # We only copy contents we recognize as tags
+            if regex_tag.match(shape.text):
+                shape.name = shape.text
```

Comment:
```suggestion
                shape.name = regex_tag.group(1)
```
We should write exactly the tag into the `name`.

Not quite sure if this is better with or without curly braces. Pro-with: It's explicit and there is little chance of mixing this up with existing names of non-placeholder shapes. Con-with: It's more verbose and more cumbersome to match in `_find_shapes`.

Reply:
I would leave the curly braces. How else can we distinguish it from names not meant to be tags?